calico-enterprise/network-policy/application-layer-policies/alp-tutorial.mdx

```diff
@@ -0,0 +1,118 @@
+---
+description: Learn how you can apply ALP to your workloads and control ingress traffic.
+---
+
+# Enforce application layer policies on your apps (tutorial)
+
+This tutorial demonstrates how to use {{prodname}} application layer policy to mitigate some common threats in a sample banking application.
+Assume that the application is installed in your {{prodname}} cluster and that the following application pods are running. 
+
+```
+NAME                        READY     STATUS    RESTARTS   AGE
+customer-2809159614-qqfnx   3/3       Running   0          21h
+database-1601951801-m4w70   3/3       Running   0          21h
+summary-2817688950-g1b3n    3/3       Running   0          21h
+```
+
+Here, we have three microservices - customer, database, and summary. Only pods that need access to a service should get it.
+The customer web service does not need, and should not have direct access to the backend database. The customer web service needs to 
+directly interact with clients outside the cluster, some of whom may be malicious. 
+
+Imagine what would happen if an attacker were to gain control of the customer web pod in our
+application. Let's simulate this by executing a remote shell inside that pod.
+
+    kubectl exec -ti customer-<fill in pod ID> -c customer -- bash
+
+Notice that from here, we get direct access to the backend database. For example, we can list all the entries in the database like this:
+
+    curl http://database:2379/v2/keys?recursive=true | python -m json.tool
+
+(Piping to `python -m json.tool` nicely formats the output.)
+
+
+## Applying a network policy
+
+With a {{prodname}} policy, you can mitigate risks to the banking application.
```

Comment:
In this step, we get the application layer policy YAML and apply it.

Reply:
@bmckercher123 Do you want me to add these two sentences?